library/core/src/intrinsics.rs

```diff
@@ -1173,133 +1173,133 @@ extern "rust-intrinsic" {
     /// Returns the square root of an `f32`
     ///
     /// The stabilized version of this intrinsic is
-    /// [`std::f32::sqrt`](../../std/primitive.f32.html#method.sqrt)
+    /// [`f32::sqrt`]
```

Comment:
nit: this is now short enough to go on one line :) do you mind moving `[f32::sqrt]` up to line 1175 so it takes less space? Same for all the rest of the links you changed.

Reply:
No problem!